utils/networking.js

```diff
@@ -1,15 +1,11 @@
 import axios from 'axios';
-import { v4 as uuidv4 } from 'uuid'
 
 // Modify to test a local server
 // const API_BASE = 'http://localhost:2354';
 const API_BASE = 'https://node-1.zesty.market'
 const METRICS_ENDPOINT = API_BASE + '/api/v1/metrics'
 
 const AD_ENDPOINT = 'https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-graph-rinkeby'
-
-const sessionId = uuidv4();
```

Comment:
Thanks for removing an appendage for us 🙏🏼

Reply:
We still need this when we reimplement the metrics, don't we?

Reply:
I'd rather just have those lines commented out for now

Reply:
Woops, listen to this man ^